requirements.json

```diff
@@ -0,0 +1,85 @@
+{
+  "$schema": "https://raw.githubusercontent.com/DataDog/auto_inject/refs/heads/main/preload_go/cmd/library_requirements_tester/testdata/requirements_schema.json",
+  "version": 1,
+  "native_deps": {
+    "glibc": [{
+      "arch": "arm",
+      "supported": true,
+      "description": "From ubuntu xenial (16.04)",
+      "min": "2.23"
+    },{
+      "arch": "arm64",
+      "supported": true,
+      "description": "From centOS 7",
+      "min": "2.17"
+    },{
+      "arch": "x64",
+      "supported": true,
+      "description": "From centOS 7",
+      "min": "2.17"
+    },{
+      "arch": "x86",
+      "supported": true,
+      "description": "From debian jessie (8)",
+      "min": "2.19"
+    }],
+    "musl": [{
+      "arch": "arm",
+      "supported": true,
+      "description": "From alpine 3.13"
+    },{
+      "arch": "arm64",
+      "supported": true,
+      "description": "From alpine 3.13"
+    },{
+      "arch": "x64",
+      "supported": true,
+      "description": "From alpine 3.13"
+    },{
+      "arch": "x86",
+      "supported": true,
```

Comment:
In the description you say "We should support 32-bit for Alpine" - should this (and `arm`? be `false`)?

Reply:
I reverted that after posting the original PR, because we don't publish binaries for it, meaning that while some features won't work, the library itself does work so there wasn't really a reason to stop injection.

Reply:
Great, as long as you handle that in-library then there are no issues I think 👍